build.gradle.kts

```diff
@@ -27,7 +28,7 @@ allprojects {
     }
 
     checkstyle {
-        toolVersion = "8.26"
+        // toolVersion = "8.37" - the current
```

Comment:
I prefer to fix version in gradle configuration to ensure build won't suddenly fail

Reply:
restored, with comment on checking against current version of checkstyle plugin